src/test/resources/data/test_parse_protein_change.csv

```diff
@@ -0,0 +1,92 @@
+#
+# List of alterations for testing.  Each line is the following format:
+#
+# Protein Change,Expected consequence,Expected Reference Allele,Expected Variant Allele,Expected Protein Start,Expected Protein End
+#
+
+# missense variants
+N505I,missense_variant,N,I,505,505
+E323_D324delinsKN,missense_variant,,,323,324
+814_852mis,missense_variant,,,814,852
+IK744KI,missense_variant,IK,KI,744,745
+
+# inframes
+Q58_Q59insL,inframe_insertion,,,58,59
+58_Q59insL,inframe_insertion,,,58,59
+Q58_59insL,inframe_insertion,,,58,59
+58_59insL,inframe_insertion,,,58,59
+P68_C77dup,inframe_insertion,,,68,77
+P68_77dup,inframe_insertion,,,68,77
+68_C77dup,inframe_insertion,,,68,77
+68_77dup,inframe_insertion,,,68,77
+T599delinsIP,inframe_insertion,T,,599,599
+599delinsIP,inframe_insertion,,,599,599
+599delins,UNKNOWN,,,599,599
+599delins2,UNKNOWN,,,599,599
+I744KI,inframe_insertion,I,KI,744,744
+M1ext-1,inframe_insertion,,,1,1
+1ext-1,inframe_insertion,,,1,1
+1ext,inframe_insertion,,,1,1
+C359del,inframe_deletion,C,,359,359
+359del,inframe_deletion,,,359,359
+IK744A,inframe_deletion,IK,A,744,745
+1459_1468del,inframe_deletion,,,1459,1468
+A1459_D1468del,inframe_deletion,,,1459,1468
+1459_D1468del,inframe_deletion,,,1459,1468
+A1459_1468del,inframe_deletion,,,1459,1468
+D1161_S1172delinsE,inframe_deletion,,,1161,1172
+D1161_1172delinsE,inframe_deletion,,,1161,1172
+1161_S1172delinsE,inframe_deletion,,,1161,1172
+1161_1172delinsE,inframe_deletion,,,1161,1172
+
+# frame shifts
+S859Afs*12,frameshift_variant,S,,859,859
+859Afs*12,frameshift_variant,,,859,859
+S859fs*12,frameshift_variant,S,,859,859
+G314fs,frameshift_variant,G,,314,314
+314fs,frameshift_variant,,,314,314
+S330_S352fs,frameshift_variant,,,330,352
+330_S352fs,frameshift_variant,,,330,352
+S330_352fs,frameshift_variant,,,330,352
+330_352fs,frameshift_variant,,,330,352
+
+#  truncating mutations
+W143_A314trunc,feature_truncation,,,143,314
+W143_314trunc,feature_truncation,,,143,314
+143_A314trunc,feature_truncation,,,143,314
+422_605trunc,feature_truncation,,,422,605
+Truncating Mutations,feature_truncation,,,,
+
+# splice variants
+596_619splice,splice_region_variant,,,596,619
+X1429_splice,splice_region_variant,,,1429,1429
+1429_splice,splice_region_variant,,,1429,1429
+M1I,start_lost,M,I,1,1
+1I,start_lost,,I,1,1
+M1?,start_lost,M,?,1,1
+A149*,stop_gained,A,*,149,149
+149*,stop_gained,,*,149,149
+*149*,stop_lost,*,*,149,149
+
+# extension
+*757Kext*36,stop_lost,*,,757,757
+757Kext*36,stop_lost,*,,757,757
+*757ext*36,stop_lost,*,,757,757
+*757Kext*,stop_lost,*,,757,757
+*757Kext*?,stop_lost,*,,757,757
```

Comment:
Do you have some examples for `M?1ext(-[0-9]+)?` ?

Reply:
Good catch. We don't. I'm digging more details on this one, will update.